new-findings.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
```

Comment:
Consider using `set -x` so that each command will be printed as the script runs. (This would avoid the need for most of the `echo`s).

Reply:
Done.